src/components/HendelseTypeFilter.tsx

```diff
@@ -77,14 +85,28 @@ const Container = styled.div`
   margin-bottom: 1rem;
 `;
 
+const getHendelsetekster = (visDialogmotekandidat: boolean) => {
+  if (visDialogmotekandidat) {
+    return HendelseTekster;
+  }
+  const { DIALOGMOTEKANDIDAT, ...hendelseTekster } = HendelseTekster;
```

Comment:
Litt dumt med warning her, men må vel bruke https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings for å unngå det 🤔

Reply:
Ja det er litt dumt ja, men er kanskje levbart om det er midlertidig?

Reply:
Joda, ikke noe stress det